
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../../project/results/">
      
      
      
      <link rel="icon" href="../../img/idec-wiki-logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.4.5">
    
    
      
        <title>Protocols - iDEC 2023 | Tidetron</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.6a10b989.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.356b1318.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="light-blue" data-md-color-accent="blue">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#protocols" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="iDEC 2023 | Tidetron" class="md-header__button md-logo" aria-label="iDEC 2023 | Tidetron" data-md-component="logo">
      
  <img src="../../img/idec-wiki-logo-white-bg.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            iDEC 2023 | Tidetron
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Protocols
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/idec-teams/2023_Tidetron" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="iDEC 2023 | Tidetron" class="md-nav__button md-logo" aria-label="iDEC 2023 | Tidetron" data-md-component="logo">
      
  <img src="../../img/idec-wiki-logo-white-bg.png" alt="logo">

    </a>
    iDEC 2023 | Tidetron
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/idec-teams/2023_Tidetron" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
    <li class="md-nav__item">
      <a href="../../team/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Team
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Project
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Project
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../project/background/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Background
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../project/methods/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Methods
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../project/results/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Results
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" checked>
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Documentation
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Documentation
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Protocols
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Protocols
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#1-processing-of-rna-seq-data" class="md-nav__link">
    1. Processing of RNA-Seq Data
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#2-protein-tertiary-structure-prediction-and-mutation-analysis" class="md-nav__link">
    2. Protein Tertiary Structure Prediction and Mutation Analysis
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#3-collagen-and-elastin-purification" class="md-nav__link">
    3. Collagen and Elastin Purification
  </a>
  
    <nav class="md-nav" aria-label="3. Collagen and Elastin Purification">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#a-chromatography-free-purification-method" class="md-nav__link">
    a. Chromatography-free purification method
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#4-artp-pance-for-screening-intein-and-t7-rnap-mutants-in-vivo" class="md-nav__link">
    4. ARTP-PANCE for Screening Intein and T7 RNAP Mutants In Vivo
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#5-crispr-associated-transposon-protocol" class="md-nav__link">
    5. CRISPR-Associated Transposon Protocol
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#6-tadr-fads-for-screening-proline-hydroxylase-mutants" class="md-nav__link">
    6. TADR-FADS for Screening Proline Hydroxylase Mutants
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#7-microfluid-chip-and-device-preparation" class="md-nav__link">
    7. Microfluid Chip and Device Preparation
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#8-activity-assay-for-proline-hydroxylase" class="md-nav__link">
    8. Activity Assay for Proline Hydroxylase
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#9-collagen-and-elastin-synthesis-via-cfps-reaction" class="md-nav__link">
    9. Collagen and Elastin Synthesis via CFPS Reaction
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#10-proline-hydroxylase-mutant-screening-via-microfluidic-chips" class="md-nav__link">
    10. Proline Hydroxylase Mutant Screening via Microfluidic Chips
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#1-processing-of-rna-seq-data" class="md-nav__link">
    1. Processing of RNA-Seq Data
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#2-protein-tertiary-structure-prediction-and-mutation-analysis" class="md-nav__link">
    2. Protein Tertiary Structure Prediction and Mutation Analysis
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#3-collagen-and-elastin-purification" class="md-nav__link">
    3. Collagen and Elastin Purification
  </a>
  
    <nav class="md-nav" aria-label="3. Collagen and Elastin Purification">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#a-chromatography-free-purification-method" class="md-nav__link">
    a. Chromatography-free purification method
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#4-artp-pance-for-screening-intein-and-t7-rnap-mutants-in-vivo" class="md-nav__link">
    4. ARTP-PANCE for Screening Intein and T7 RNAP Mutants In Vivo
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#5-crispr-associated-transposon-protocol" class="md-nav__link">
    5. CRISPR-Associated Transposon Protocol
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#6-tadr-fads-for-screening-proline-hydroxylase-mutants" class="md-nav__link">
    6. TADR-FADS for Screening Proline Hydroxylase Mutants
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#7-microfluid-chip-and-device-preparation" class="md-nav__link">
    7. Microfluid Chip and Device Preparation
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#8-activity-assay-for-proline-hydroxylase" class="md-nav__link">
    8. Activity Assay for Proline Hydroxylase
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#9-collagen-and-elastin-synthesis-via-cfps-reaction" class="md-nav__link">
    9. Collagen and Elastin Synthesis via CFPS Reaction
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#10-proline-hydroxylase-mutant-screening-via-microfluidic-chips" class="md-nav__link">
    10. Proline Hydroxylase Mutant Screening via Microfluidic Chips
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<h1 id="protocols">Protocols</h1>
<h3 id="1-processing-of-rna-seq-data">1. <strong>Processing of RNA-Seq Data</strong></h3>
<p><strong>Materials</strong>:
- Computer with installed R studio and relevant bioinformatics packages.
- Human tissue RNA-seq data from specified databases.</p>
<p><strong>Procedure</strong>:
1. <strong>Preparation</strong>: Install necessary packages and libraries on R studio. Ensure stable internet connectivity for data download.
2. <strong>Data Download</strong>:
   - Navigate to the respective database web pages.
   - Select the desired datasets and initiate download.
   - Store data in a specified folder for easy access.
3. <strong>Processing</strong>:
   - Load the data into R studio.
   - Use the <code>DESeq2</code> package (or similar) for differential expression analysis.
   - Filter out genes with low counts.
   - Normalize the data.
4. <strong>Analysis</strong>:
   - Generate heatmaps, PCA plots, and other relevant plots to visualize gene expression patterns.
   - Save and catalog results.</p>
<p><strong>Specific Conditions</strong>:
- Ensure the computer has sufficient memory and processing power for large datasets.</p>
<hr />
<h3 id="2-protein-tertiary-structure-prediction-and-mutation-analysis">2. <strong>Protein Tertiary Structure Prediction and Mutation Analysis</strong></h3>
<p><strong>Materials</strong>:
- Computer with internet access.
- AlphaFold2 platform access.
- UCSF Chimera software.
- DNA sequences for the target proteins.</p>
<p><strong>Procedure</strong>:
1. <strong>Protein Prediction</strong>:
   - Navigate to the Bkunyun Platform and upload the DNA/protein sequence.
   - Initiate AlphaFold2 prediction and wait for results.
2. <strong>Structural Analysis</strong>:
   - Download the resulting PDB files.
   - Open the PDB files in UCSF Chimera.
   - Visualize, compare, and analyze structures.
3. <strong>Mutation Analysis</strong>:
   - Highlight and label mutation sites on the structures using UCSF Chimera tools.
   - Predict potential functional changes due to mutations.</p>
<p><strong>Specific Conditions</strong>:
- Ensure a stable internet connection for online platforms.
- Ensure sufficient storage space for saving PDB and other files.</p>
<hr />
<h3 id="3-collagen-and-elastin-purification">3. <strong>Collagen and Elastin Purification</strong></h3>
<h4 id="a-chromatography-free-purification-method"><strong>a. Chromatography-free purification method</strong></h4>
<p><strong>Materials</strong>:
- BL21 (DE3) strain with appropriate plasmid.
- PBS (pH 9.0, 0.5 M NaCl).
- 10 KDa &amp; 1 KDa ultrafiltration tubes.
- HPLC system.</p>
<p><strong>Procedure</strong>:
1. <strong>Cultivation</strong>:
   - Inoculate a fresh culture of BL21 in LB medium (pH 8.5) supplemented with 50 mg/L kanamycin.
   - Incubate at 37°C, 200 rpm until OD600 ~0.6.
   - Add 0.3 mM IPTG to induce protein expression and incubate overnight.
2. <strong>Protein Extraction</strong>:
   - Harvest cells by centrifugation at 4,000 rpm, 15 minutes, 4°C.
   - Resuspend the cell pellet in 50 mL PBS.
   - Lyse cells using high-pressure homogenization.
   - Centrifuge lysed cells at 10,000 rpm, 30 minutes, 4°C to remove cell debris.
3. <strong>Ultrafiltration</strong>:
   - Pass supernatant through a 10 KDa ultrafiltration tube to remove proteins &gt;10 KDa.
   - Adjust pH of filtrate to 6.5 using 1 M HCl.
   - Incubate at 30°C overnight for potential peptide cleavage.
   - Pass the solution through a 1 KDa ultrafiltration tube.
4. <strong>Quality Control</strong>:
   - Analyze filtrate using HPLC to assess peptide purity and concentration.</p>
<p><strong>Specific Conditions</strong>:
- All steps involving cells and proteins should be done on ice or at 4°C, unless otherwise stated.
- Use sterile techniques to prevent contamination.</p>
<hr />
<h3 id="4-artp-pance-for-screening-intein-and-t7-rnap-mutants-in-vivo">4. <strong>ARTP-PANCE for Screening Intein and T7 RNAP Mutants In Vivo</strong></h3>
<p><strong>Materials</strong>:
- Primary M13 phage for GyrA or T7 RNAP.
- S1030 strain with pJC175e plasmid.
- DNA-LAMP method kit.
- LB medium, appropriate antibiotics.</p>
<p><strong>Procedure</strong>:
1. <strong>Phage Addition</strong>:
   - Inoculate a fresh culture of S1030 strain containing the pJC175e plasmid.
   - At early log phase, add the primary M13 phage.
   - Incubate to allow for bacterial-phage interactions.
2. <strong>ARTP Mutation</strong>:
   - Centrifuge and re-suspend bacterial cells in fresh LB medium.
   - Expose to ARTP treatment in controlled settings for specified durations (e.g., 30s, 120W).
   - Dilute treated cells in fresh LB medium.
3. <strong>Phage Titer Estimation</strong>:
   - Use the DNA-LAMP method to periodically measure phage titers.
   - When phage titer reaches 10^6 CFU/mL, proceed to the next step.
4. <strong>Phage Collection</strong>:
   - Centrifuge culture and collect the supernatant containing the M13 phage.</p>
<p><strong>Specific Conditions</strong>:
- Use sterile conditions throughout to prevent contamination.
- The ARTP mutagenesis equipment should be calibrated to provide the right exposure intensity.
- Monitor cultures for any signs of contamination or cell death.</p>
<hr />
<h3 id="5-crispr-associated-transposon-protocol">5. <strong>CRISPR-Associated Transposon Protocol</strong></h3>
<p><strong>Materials</strong>:
- BL21 strain containing specified plasmids.
- LB medium with appropriate antibiotics.
- PCR equipment, reagents, and primers.
- Sanger sequencing setup.</p>
<p><strong>Procedure</strong>:
1. <strong>Strain Induction</strong>:
   - Inoculate BL21 strain with the necessary plasmids in LB medium with required antibiotics.
   - Grow until OD600 reaches 0.6.
   - Induce with 0.2 mM IPTG and continue incubation overnight.
2. <strong>Isolation</strong>:
   - Plate culture on LB plates with antibiotics to isolate monoclonal strains.
   - Grow plates until colonies are visible.
3. <strong>Identification</strong>:
   - Use bacterial PCR to identify strains with the desired insert fragments.
   - Confirm insertions through Sanger sequencing.</p>
<p><strong>Specific Conditions</strong>:
- Use a controlled incubator set at 37°C for bacterial growth.
- Maintain sterility throughout to avoid contamination.
- Optimize PCR conditions for specific primers and target regions.</p>
<hr />
<h3 id="6-tadr-fads-for-screening-proline-hydroxylase-mutants">6. <strong>TADR-FADS for Screening Proline Hydroxylase Mutants</strong></h3>
<p><strong>Materials</strong>:
- BL21 strain with specific genomic modifications and plasmids.
- Fluorescent microplate reader.
- LB medium and appropriate antibiotics.</p>
<p><strong>Procedure</strong>:
1. <strong>Strain Transformation</strong>:
   - Transform BL21 with the desired plasmid.
   - Grow transformed cells in LB medium with appropriate antibiotics until OD600 ~0.6.
2. <strong>Induction</strong>:
   - Add 0.5 mM IPTG to the culture.
   - Continue overnight incubation.
3. <strong>Fluorescence Measurement</strong>:
   - Measure GFP fluorescence intensity of the cultures using a fluorescent microplate reader.</p>
<p><strong>Specific Conditions</strong>:
- Maintain cultures at 37°C, 200 rpm throughout.
- Calibration of the microplate reader is essential for accurate fluorescence readings.</p>
<hr />
<h3 id="7-microfluid-chip-and-device-preparation">7. <strong>Microfluid Chip and Device Preparation</strong></h3>
<p><strong>Materials</strong>:
- PDMS
- HFE7500 fluorinated oil.
- Microscopy and imaging setup with lasers and cameras.</p>
<p><strong>Procedure</strong>:
1. <strong>Chip Fabrication</strong>:
   - Create microfluidic patterns using PDMS.
   - Allow PDMS to cross-link and cure.
   - Punch the necessary ports and channels.
2. <strong>Assembly</strong>:
   - Attach the PDMS structure to a glass slide.
   - Apply a hydrophobic surface coating.
   - Fill electrode holes as necessary.
3. <strong>Droplet Generation</strong>:
   - Use a dropmaker chip with fluorinated oil to generate droplets.
   - Employ imaging equipment to monitor and manipulate droplets.</p>
<p><strong>Specific Conditions</strong>:
- PDMS curing should be done under controlled temperature conditions for consistency.
- Maintain a clean environment to ensure dust or contaminants do not interfere with microfluidic operations.</p>
<hr />
<h3 id="8-activity-assay-for-proline-hydroxylase">8. <strong>Activity Assay for Proline Hydroxylase</strong></h3>
<p><strong>Materials</strong>:
- Bacterial Protein Extraction Kit.
- Hydroxyproline (HYP) content detection kit.</p>
<p><strong>Procedure</strong>:
1. <strong>Protein Extraction</strong>:
   - Extract total proteins from E. coli using the extraction kit.
2. <strong>Quantification</strong>:
   - Measure protein concentration using a BCA protein quantification kit.
3. <strong>Activity Assessment</strong>:
   - Determine proline hydroxylation levels using the HYP content detection kit.</p>
<p><strong>Specific Conditions</strong>:
- All extraction steps should be carried out at 4°C or on ice to maintain protein stability.
- Ensure that all reagents, especially those in the HYP detection kit, are at the appropriate temperature and freshly prepared.
- Control samples (non-hydroxylated proline samples) should be run alongside experimental samples for accurate comparison.</p>
<hr />
<h3 id="9-collagen-and-elastin-synthesis-via-cfps-reaction">9. <strong>Collagen and Elastin Synthesis via CFPS Reaction</strong></h3>
<p><strong>Materials</strong>:
- 20 standard amino acids, glycine, proline, 2-oxoglutarate, etc.
- Cell extract from a high yield source.
- Target plasmid or DNA encoding collagen/elastin sequences.
- Necessary buffers and cofactors for in vitro transcription and translation.</p>
<p><strong>Procedure</strong>:
1. <strong>Preparation</strong>:
   - Thaw all reagents on ice.
   - Prepare a master mix containing all the components except the cell extract and DNA.
2. <strong>Reaction Setup</strong>:
   - In a sterile reaction tube, mix the master mix, cell extract, and target DNA/plasmid.
   - Ensure thorough mixing without introducing bubbles.
3. <strong>Incubation</strong>:
   - Place the reaction tube in a controlled incubator set at 37°C.
   - Allow the reaction to proceed for 8 hours.
4. <strong>Analysis</strong>:
   - After incubation, analyze the reaction mixture for synthesized collagen/elastin using SDS-PAGE or Western Blot.</p>
<p><strong>Specific Conditions</strong>:
- Ensure all components are fresh and uncontaminated. This is crucial for the efficiency of the CFPS system.
- Maintain a sterile environment during the setup to prevent external protein or RNA contamination.
- Optimal temperatures for the CFPS system might vary depending on the specific cell extract source. Adjust accordingly.</p>
<hr />
<h3 id="10-proline-hydroxylase-mutant-screening-via-microfluidic-chips">10. <strong>Proline Hydroxylase Mutant Screening via Microfluidic Chips</strong></h3>
<p><strong>Materials</strong>:
- Microfluidic chips prepared from PDMS.
- HFE7500 fluorinated oil.
- Culture of strains containing the P4Hc mutant library.
- Necessary buffers and reagents for droplet generation.</p>
<p><strong>Procedure</strong>:
1. <strong>Strain Preparation</strong>:
   - Grow the strains containing the P4Hc mutant library in LB medium until OD600 reaches ~0.6.
   - Dilute the culture for use in droplet generation.
2. <strong>Droplet Generation</strong>:
   - Set up the microfluidic chip on the appropriate device.
   - Introduce the bacterial culture and HFE7500 fluorinated oil into the chip to generate droplets.
3. <strong>Droplet Analysis</strong>:
   - Using the attached imaging system, monitor the droplets for desired phenotypes or activities.
   - Sort and collect droplets of interest for further analysis.</p>
<p><strong>Specific Conditions</strong>:
- Ensure consistent flow rates for both the bacterial culture and the fluorinated oil to generate uniform droplets.
- Calibrate the imaging system for accurate detection and analysis of droplets.
- Maintain a clean setup to prevent clogging or contamination within the microfluidic channels.</p>





                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      Copyright &copy; 2023 iDEC. CC-BY-4.0 | The International Directed Evolution Competition (iDEC) is a Scottish Charitable Incorporate Organisation (SCIO), registered in Scotland, with charity number：SC050826
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.sections"], "search": "../../assets/javascripts/workers/search.f886a092.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../../assets/javascripts/bundle.aecac24b.min.js"></script>
      
        <script src="../../javascripts/config.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/tablesort/5.2.1/tablesort.min.js"></script>
      
        <script src="../../javascripts/tables.js"></script>
      
    
  </body>
</html>